test/common/testlib.py

```diff
@@ -2090,6 +2090,19 @@ def write_file(self, path: str, content: str, append: bool = False, owner: Optio
         m = self.machine
         self.restore_file(path, post_restore_action=post_restore_action)
         m.write(path, content, append=append, owner=owner, perm=perm)
+        
```

Comment:
only one empy line between methods, test/static-code will complain otherwise. Same is true for the next two paragraphs.